src/coreclr/vm/jitinterface.h

```diff
@@ -1157,8 +1157,9 @@ CORJIT_FLAGS GetDebuggerCompileFlags(Module* pModule, CORJIT_FLAGS flags);
 
 bool __stdcall TrackAllocationsEnabled();
 
-FCDECL0(INT64, GetJittedBytes);
-FCDECL0(INT32, GetJittedMethodsCount);
+FCDECL1(int64_t, GetCompiledILBytes, bool currentThread);
```

Comment:
The bool arguments should be `CLR_BOOL`. C++ bool does not always match managed bool.

https://github.com/dotnet/runtime/blob/main/docs/design/coreclr/botr/corelib.md